### PR TITLE
VIH-10743 select interpreter type

### PR DESCRIPTION
**Pull request by shaed-parkar**
### Jira link (if applicable)

VIH-10743

### Change description ###

Select the interpretation type for non interpreter participants/endpoints
This is hidden behind the feature flag: `interpreter-enhancements`

image

image

image

image

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/models/add-judicial-member.model.ts [hunk 1]
```diff
@@ -31,6 +33,9 @@ export class JudicialMemberDto {
         dto.optionalContactEmail = response.optional_contact_email;
         dto.roleCode = response.role_code as JudicaryRoleCode;
         dto.displayName = response.display_name;
+
+        // todo: map from response when API is implemented
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/models/add-judicial-member.model.ts:37]: there's a TODO here, is that part of a separate ticket?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts [hunk 2]
```diff
@@ -29,6 +30,7 @@ export class ParticipantModel {
     is_staff_member?: boolean;
     contact_email?: string;
     isJudiciaryMember?: boolean;
+    interpretationLanguage: InterpreterSelectedDto;
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts:33]: we have a mix of snake_case and camelCase conventions for these property names, perhaps we should stick with one or the other. Maybe snake_case as that is the majority?

**Review by oliver-scott — COMMENTED** [review]
looks really good, just a couple of comments

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/models/add-judicial-member.model.ts (hunk 1 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/models/add-judicial-member.model.ts:37]: Yup, there's a ticket for the API changes that need to update the contracts too

File: AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts (hunk 2 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts]: would be nice to switch to camelCase as the rest of the repo but we'll save that for another day

**Review by oliver-scott — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Pull request merged**